app/src/main/java/org/gateshipone/odyssey/activities/OdysseyMainActivity.java

```diff
@@ -120,16 +120,14 @@
 
     public final static String MAINACTIVITY_INTENT_EXTRA_BACKTOSETTINGS = "org.gateshipone.odyssey.backtosettings";
 
-    public final static String MAINACTIVITY_INTENT_EXTRA_BACKTOSETTINGS_VALUE = "org.gateshipone.odyssey.backtosettings.value";
-
     private Uri mSentUri;
 
     private boolean mShowNPV = false;
 
-    private boolean mBackToSettings = false;
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        boolean mBackToSettings = false;
```

Comment:
This variable does not follow the naming convention. Should be named backToSettings